docs/source/model_design/vivarium_features/automated_v_and_v/index.rst

```diff
@@ -0,0 +1,245 @@
+
+We can measure the statistical power of our hypothesis tests.
+To do this in a flexible way that does not require an effect size estimate specific to each hypothesis,
+we can report the range of simulation values we would have an 80% chance of detecting (power = 0.8).
+This wouldn't affect whether the fuzzy check passes or fails; it would be used for human inspection to
+get a sense of whether the computational resources allocated to the automated V&V are sufficient.
+Note that the power calculations do not depend on what is actually observed in the simulation, unless dynamic behavior
+changes our sample size.
+Therefore, in most cases power calculation results should not need to be re-inspected every time there is an update to the simulation.
+
+Value types
++++++++++++
+
+For now, we have only investigated methods for fuzzy checking proportions and rates.
+This is sufficient for a proof-of-concept implementation, which applies fuzzy checking
+to migration rates in the PRL simulation.
+Presumably, other types of values could be checked using the appropriate hypothesis tests.
+
+Proportions and rates
+~~~~~~~~~~~~~~~~~~~~~
+
+In our discrete-time simulations, rates can be seen as equivalent to proportions.
+On each time step, a given event happens to some proportion of the population at risk.
+
+The proportion we observe in the simulation is the result of some number of independent Bernoulli trials,
+one for each simulant at risk.
+Usually, in our simulations, the probability associated with each simulant/trial varies only according
+to some categorical risk factors, which means that within each combination of categories,
+the probability is the same for all simulants and the number of events has a binomial distribution.
+Therefore, a `binomial test <https://sites.utexas.edu/sos/guided/inferential/categorical/univariate/binomial/>`_
+can determine the p-value of the simulation result in that group, which is the probability
+of observing a result equally or less likely, if the simulation's value is correct.
+We use two-tailed tests in order to catch errors in either direction.
+
+When simulant-level probabilities of an event vary within a group (for example, if there is a continuous risk factor
+of the event), the Bernoulli trials are independent but not identically distributed.
+The number of events observed has a `Poisson binomial <https://en.wikipedia.org/wiki/Poisson_binomial_distribution>`_
+distribution.
+This distribution has the same mean and **lower** variance, relative to a binomial distribution where each trial
+has the mean probability.
+This simple binomial distribution can be used as an approximation, allowing the use of a binomial test in this situation as well;
+due to the variance property, this approximation will increase the specificity of the fuzzy test to be higher than
+necessary to achieve the specified family-wise error rate.
+This increase in specificity will cause a decrease in sensitivity.
```

Comment:
Similarly here, is there a way to show how the use of binomial/poisson would change the specificity/sensitivity visually?